apps/wallet/src/ui/app/pages/approval-request/SignMessageRequest.tsx

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { fromB64 } from '@mysten/sui.js';
+import { useMemo } from 'react';
+
+import { UserApproveContainer } from '../../components/user-approve-container';
+import { useAppDispatch } from '../../hooks';
+import { respondToTransactionRequest } from '../../redux/slices/transaction-requests';
+import { Heading } from '../../shared/heading';
+import { Text } from '../../shared/text';
+import { type SignMessageApprovalRequest } from '_payloads/transactions/ApprovalRequest';
+
+export type SignMessageRequestProps = {
+    request: SignMessageApprovalRequest;
+};
+
+export function SignMessageRequest({ request }: SignMessageRequestProps) {
+    const message = useMemo(() => {
+        return new TextDecoder().decode(fromB64(request.tx.message));
```

Comment:
I imagine there's a non-zero chance this causes useless output (in the event raw bytes are signed), might want a way to see the raw bytes being signed.

Reply:
https://mysten.atlassian.net/browse/APPS-612